src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -172,7 +184,7 @@ export class CodeComponent extends AngularDisposable implements OnInit, OnChange
 		if (this.destroyed) {
 			return;
 		}
-		this.createEditor();
+		this.createEditor().catch((err) => console.error(err));
```

Comment:
Don't use console.error directly, use the vscode onUnexpectedError function instead (that'll output to the console as well)

Also it'd be nice to add the void return type for this just so we're explicit in what it's returning

Reply:
I have seen a logservice being use in other places in this class, going to follow that for here. But I'll make sure to use onUnexpectedError instead of console.error. Thanks.

Reply:
Yes, onUnexpectedError is preferred over logservice (onUnexpectedError calls the logservice itself so it'll still result in the same thing happening)